models/profiles.yaml

```diff
@@ -27,7 +27,29 @@ models:
             select: "{{macro_datediff('{{user.Var(\"max_timestamp_bw_tracks_pages\")}}')}}"
             dependencies:
               - max_timestamp_bw_tracks_pages
-
+        # New boolean feature based on days_since_last_seen
+        - entity_var:
+            name: has_seen_recently
+            select: CASE 
+                    WHEN RAND() > 0.99 THEN null
```

Comment:
How many rows do we have in the input table? this creates only 1% nulls, and if the input table is small (<100 rows), we may end up with zero nulls. can you ensure we have decent number of rows there? We can probably increase the rand value to 0.9